tomato.lua

```diff
@@ -181,7 +150,7 @@ minetest.register_abm({
 })
 
 crops.tomato_die = function(pos)
-	minetest.set_node(pos, { name = "crops:tomato_plant_6", param2 = 1 })
+	minetest.swap_node(pos, { name = "crops:tomato_plant_6", param2 = 1 })
```

Comment:
`set_node` erases the metadata, which is intentional here. This is probably wrong.